packages/quantic/cypress/fixtures/mock-result.ts

```diff
@@ -0,0 +1,59 @@
+//TODO: Refactor with ./packages/atomic/cypress/utils/mock-result.ts
```

Comment:
What kind of refactoring do we want to do here?

Reply:
Eventually have a common testing package used both by Atomic & Quantic Cypress tests.

Like, a result is a result, it ain't gonna change in Atomic or Quantic.

We don't want either to exposes it to the customer, because it'll be a hassle to support testing utilities as product